### ISSUE TITLE
set up a hook to send built templates over to mezzanine

### ISSUE THREAD

**Issue by Pomax** [mozilla/network#241]
migrated from https://github.com/mozilla/network-api/issues/89

There's a few places this could be done, any post-build moment should work for this purpose. We basically need to do a push into the mezzanine repo for known files.

For testing, this should probably go into its own dir, so we can do manual verification, with the mezzanine settings updated to be aware of this new template location.

From @simonwex:
>@cadecairos had discussed setting up template-loading in mezzanine to possibly load from S3. This feels lower priority than the other things in this milestone. If it needs pushing out, I think that's fine.

**Comment by Pomax** [comment]
In the interest of doing what needs to be done, with what is nice to be done in follow-up, let's push this out for now - we're not updating the templates so frequently that it's a hassle to just copy over manually, and copying over manually is a guaranteed unblock for making sure the pug-generated templates align with what Mezzanine wants.

**Comment by Pomax** [comment]
We will be doing this manually for the immediate future in the interest of having a working, but not necessarily fully automated, implementation.

**Comment by alanmoo** [comment]
@cadecairos Can you confirm this is already done?

**Comment by cadecairos** [comment]
This is done: https://jenkins.mofoprod.net/job/copy-network-templates-to-mezzanine/
